demos/html5/froala2/test/e2e-insert-formula.test.js

```diff
@@ -34,6 +34,7 @@ describe('Insert Formula. TAG = Insert',
 
     // Execute before each test of the file to open the demo page
     beforeEach(async () => {
+      jest.setTimeout(10000); // Large timeouts seem to be necessary. Default timeout to 5000ms
```

Comment:
I'd make sure this 10' wait is absolute necessary.

Ideally tests should execute as fast as possible.